packages/manager/cypress/e2e/core/images/create-image.spec.ts

```diff
@@ -84,4 +120,136 @@ describe('create image (e2e)', () => {
+    });
+
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetRegions(mockRegions).as('getRegions');
+    mockGetLinodes(mockLinodes).as('getLinodes');
+
+    // intercept request
+    cy.visitWithLogin('/images/create');
+    cy.wait([
+      '@getFeatureFlags',
+      '@getClientStream',
+      '@getAccount',
+      '@getLinodes',
+      '@getRegions',
+    ]);
+
+    // Find the Linode select and open it
+    cy.findByLabelText('Linode')
+      .should('be.visible')
+      .should('be.enabled')
+      .should('have.attr', 'placeholder', 'Select a Linode')
+      .click();
+
+    // Select the Linode
+    ui.autocompletePopper
+      .findByTitle(mockLinodes[0].label)
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    // Check if notice is visible
+    cy.get(`[data-testid=${encryptionCaveatNoticeTestId}]`).should('exist');
+  });
+
+  it('does not display a notice informing user that Images are not encrypted if the LDE feature is disabled', () => {
+    // Mock feature flag -- @TODO LDE: Remove feature flag once LDE is fully rolled out
+    mockAppendFeatureFlags({
+      linodeDiskEncryption: makeFeatureFlagData(false),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    // Mock responses
+    const mockAccount = accountFactory.build({
+      capabilities: ['Linodes', 'Disk Encryption'],
+    });
+
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetRegions(mockRegions).as('getRegions');
+    mockGetLinodes(mockLinodes).as('getLinodes');
+
+    // intercept request
+    cy.visitWithLogin('/images/create');
+    cy.wait([
+      '@getFeatureFlags',
+      '@getClientStream',
+      '@getAccount',
+      '@getLinodes',
+      '@getRegions',
+    ]);
+
+    // Find the Linode select and open it
+    cy.findByLabelText('Linode')
+      .should('be.visible')
+      .should('be.enabled')
+      .should('have.attr', 'placeholder', 'Select a Linode')
+      .click();
+
+    // Select the Linode
+    ui.autocompletePopper
+      .findByTitle(mockLinodes[0].label)
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    // Check if notice is visible
+    cy.get(`[data-testid=${encryptionCaveatNoticeTestId}]`).should('not.exist');
+  });
+
+  it('does not display a notice informing user that Images are not encrypted if the selected linode is in an Edge region', () => {
+    // Mock feature flag -- @TODO LDE: Remove feature flag once LDE is fully rolled out
+    mockAppendFeatureFlags({
+      linodeDiskEncryption: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    // Mock responses
+    const mockAccount = accountFactory.build({
+      capabilities: ['Linodes', 'Disk Encryption'],
+    });
+
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetRegions(mockRegions).as('getRegions');
+    mockGetLinodes(mockLinodes).as('getLinodes');
```

Comment:
@dwiley-akamai It looks like Cloud fires off a request to get the details for the individual Linode that was selected, and uses the region ID from that response when it calls `getIsEdgeRegion`.

Importing the `mockGetLinodeDetails` util and calling that with the data for `mockLinodes[1]` gets the test passing!

```suggestion
    mockGetLinodes(mockLinodes).as('getLinodes');
    mockGetLinodeDetails(mockLinodes[1].id, mockLinodes[1]);
```